File: tests/Generator/resource/go/default/news.go

// An general news entry
type News struct {
    Config *Meta `json:"config"`
    InlineConfig map[string]string `json:"inlineConfig"`
    MapTags map[string]string `json:"mapTags"`
    MapReceiver map[string]Author `json:"mapReceiver"`
    Tags []string `json:"tags"`
    Receiver []Author `json:"receiver"`
    Read bool `json:"read"`
    Author *Author `json:"author"`
    Meta *Meta `json:"meta"`
    SendDate string `json:"sendDate"`
    ReadDate string `json:"readDate"`
    Price float64 `json:"price"`
    Rating int `json:"rating"`
    Content string `json:"content"`
    Question string `json:"question"`
    Version string `json:"version"`
    CoffeeTime string `json:"coffeeTime"`
    Captcha string `json:"g-recaptcha-response"`
    MediaFields string `json:"media.fields"`
    Payload any `json:"payload"`
}

File: tests/Generator/resource/swift/default/News.swift
// An general news entry
class News: Codable {
    var config: Meta?
    var inlineConfig: Dictionary<String, String>?
    var mapTags: Dictionary<String, String>?
    var mapReceiver: Dictionary<String, Author>?
    var tags: Array<String>?
    var receiver: Array<Author>?
    var data: Array<Array<Float>>?
    var read: Bool?
    var author: Author
    var meta: Meta?
    var sendDate: Date?
    var readDate: Date?
    var price: Float?
    var rating: Int?
    var content: String
    var question: String?
    var version: String?
    var coffeeTime: String?
    var captcha: String?
    var mediaFields: String?
    var payload: Any?

    enum CodingKeys: String, CodingKey {
        case config = "config"
        case inlineConfig = "inlineConfig"
        case mapTags = "mapTags"
        case mapReceiver = "mapReceiver"
        case tags = "tags"
        case receiver = "receiver"
        case data = "data"
        case read = "read"
        case author = "author"
        case meta = "meta"
        case sendDate = "sendDate"
        case readDate = "readDate"
        case price = "price"
        case rating = "rating"
        case content = "content"
        case question = "question"
        case version = "version"
        case coffeeTime = "coffeeTime"
        case captcha = "g-recaptcha-response"
        case mediaFields = "media.fields"
        case payload = "payload"
    }
}

